src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -132,17 +136,21 @@ public static Tag parseTag(EntryType type, String tag) throws ParseException {
         return new Tag(type, trimmedTag);
     }
 
-    ///**
-    // * Parses {@code Collection<String> tags} into a {@code Set<Tag>}.
-    // */
-    //public static Set<Tag> parseTags(Collection<String> tags) throws ParseException {
-    //    requireNonNull(tags);
-    //    final Set<Tag> tagSet = new HashSet<>();
-    //    for (String tagName : tags) {
-    //        tagSet.add(parseTag(tagName));
-    //    }
-    //    return tagSet;
-    //}
+    /**
+     * Parses a {@code String yearMonth} into a {@code YearMonth}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code yearMonth} is invalid.
+     */
+    public static YearMonth parseYearMonth(String yearMonth) throws ParseException {
```

Comment:
I think it's better to keep this `YearMonth` implementation in favor of my previous `Month` as it's more readable.